### PR TITLE
Auto unfollow

### PR DESCRIPTION
**Pull request by jyaaan**

### CODE REVIEW

File: database.js [hunk 1]
```diff
@@ -1,5 +1,70 @@
...
+get unfollow userid
+  -query relationships table for a user for client where:
+    -client_id = clientId;
+    -followed_by = false;
+    -following = true;
+    -follow_timestamp < now - 72 hours
+    -locked = false
+  -return user id of match
+
 */
```
Comment (victormejia) [inline database.js]: You can always place notes in a `notes.txt` or `notes.md` file for reference. In projects, sometimes you'll also find a "design docs" folder.

Comment (jyaaan) [reply database.js]: Ah, alright. I'll create a directory for design notes.

File: twitter-client.js [hunk 2]
```diff
@@ -75,41 +76,87 @@ Twitter.prototype.getFollowing = function () {
...
+
+Twitter.prototype.postUnfollow = function (params) {
+  console.log('posting unfollow');
+  var path = '/friendships/destroy.json' + this.buildQS(params);
+  var url = this.baseUrl + path;
+  console.log(url);
+  // return this.doPost(url, {})
+  //   .then((result) => {
+  //     return result;
+  //   });
```
Comment (victormejia) [inline twitter-client.js:141]: 👀

**Review by victormejia — APPROVED** [review]
looks good

**Pull request merged**